tests/model_selection/test_hyperband.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import numpy as np
+import scipy.stats as stats
+import math
+
+from distributed import Client
+from distributed.utils_test import loop, cluster
+
+from dask_ml.datasets import make_classification
+from dask_ml.model_selection import HyperbandCV
+
+
+class ConstantFunction:
```

Comment:
This seems like a broadly useful test utility. It could maybe go in `dask_ml/utils.py`.

Reply:
As a followup PR, doesn't need to block this.

Reply:
It's a simple fix, and will be done much quicker than this PR will be. I've moved it.